src/util/user-home-dir.js

```diff
@@ -4,7 +4,11 @@ import ROOT_USER from './root-user.js';
 
 const path = require('path');
 
+const isNotFakeRoot = typeof process.env.FAKEROOTKEY === 'undefined';
```

Comment:
I can understand the recommendation for `Boolean(...)` but as to moving the check into `root-user.js` I am not sure if I agree. It would make the imported `ROOT_USER` contain the combination of two checks, which at least from the current naming is not directly clear what exactly it is checking.